docs/api-readme.md

```diff
@@ -0,0 +1,35 @@
+# LitElement API Documentation
+
+## Install @polymer/lit-element
+
+```
+npm install --save @polymer/lit-element
+```
+
+## Modules
+
+### lit-element
+
+[lit-element](/api/modules/_lit_element_.html)
+
+Contains LitElement base class. Import `LitElement` and `html` from this module to create a component:
+
+```js
+import { LitElement, html } from '@polymer/lit-element';
+
+class MyElement extends LitElement {
+  render() {
+      return html`<p>your template here</p>`;
+  }
+}
+
+customElements.define('my-element', MyElement);
+```
+
+Inherits API functionality from [updating-element](#updating-element).
+
+### updating-element
```

Comment:
I think we should write closer to the module you'd need to import, like `lib/updating-element.js`.

We should also show the import statement so it's really clear.

Reply:
Not quite sure what you mean. Where do you want to put the import statement, and where should `updating-element` be changed to `lib/updating-element`?

Reply:
I think this should be `### lib/updating-element.js`

And right below we should have an import example like you do for lit-element:

```js
import { UpdatingElement } from '@polymer/lib/updating-element.js';
```

Except for the main module, modules don't really have names, they have paths. "updating-element" makes it look like a module name, that you could maybe import with `import {...} from 'updating-element';` or maybe `import {...} from '@polymer/updating-element';`... if you're new to imports, npm scoped packages, etc.

Reply:
Should the import statement be 

```js
import { UpdatingElement } from '@polymer/lit-element/lib/updating-element.js';
```

?